data/iKunal-Singh/iKunal-Singh.json

```
@@ -0,0 +1,101 @@
+{
+  "name": "Kunal Singh",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**Engineering undergrad** | **Blogger** | **Music Producer** bringing imagination to life.",
+  "tags": ["Open Source", "DevOps", "Linux", "Docker", "Kubernetes"],
+  "avatar": "https://github.com/iKunal-Singh.png",
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/_ikunalsingh" },
+    { "icon": "FaGithub", "url": "https://github.com/iKunal-Singh" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/ikunalsingha" },
+    { "icon": "FaSoundCloud", "url": "https://soundcloud.com/kunal-singh-558758897" },
+  ],
+  "testimonials": [ ],
+  "links": [
+    {
+      "name": "Follow for daily Lit tweets",
+      "url": "https://twitter.com/_ikunalsingh",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow my DevOps journey",
+      "url": "https://github.com/iKunal-Singh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ikunalsingha",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check Out SoundCloud",
+      "url": "https://soundcloud.com/kunal-singh-558758897",
+      "icon": "FaSoundCloud"
+    },
+    {
+      "name": "Open Source, DevOps and Linux Blog",
+      "url": "https://ikunalsingh.hashnode.dev/",
+      "icon": "SiHashnode"
+    },
```

```suggestion
    }
```
Please remove this comma